components/Trade/Basic/index.tsx

```diff
@@ -16,7 +16,7 @@ import Error from '../Error';
 import { BigNumber } from 'bignumber.js';
 import { defaults } from '@libs/Tracer';
 // @ts-ignore
-import { Experiment, Variant } from '@marvelapp/react-ab-test';
+import { Experiment, Variant } from '@tstmkrs/nextjs-ab-test';
```

Comment:
nice

Reply:
oop was meant to remove that actually. Was having this issue with SSR. just going to ignore it for now lol

Reply:
ty